source/administration/config-settings.rst

```diff
@@ -2320,9 +2320,18 @@ This setting defines the frequency of cluster request time logging for :doc:`../
 | This feature's ``config.json`` setting is ``"ClusterLogTimeoutMilliseconds": 2000`` with whole number input.                                                         |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Searching of Posts
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If this setting is enabled, users can search messages. If it's disabled, users get an error message when they attempt to use the search box.
```

Comment:
1. Ok. But it seems a bit odd... "If the feature is too popular, turn it off." :)
2. I'm not sure about mentioning search replicas. They are only available in E20 with a cluster config.